packages/shared/src/components/EmptyScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import classed from '../lib/classed';
+import { Button } from './buttons/Button';
+
+export const EmptyScreenContainer = classed(
+  'div',
+  'flex flex-col justify-center items-center px-6 w-full max-w-screen-tablet h-screen max-h-full -mt-12',
```

Comment:
Why is the negative margin top needed?

Reply:
When it is perfectly centered, it doesn't look right in terms of spacing.

Without negative margin:
![Screen Shot 2021-11-16 at 5 15 51 PM](https://user-images.githubusercontent.com/13744167/141956991-2d9ce135-c893-4486-a821-de5f4b0578f8.png)
![Screen Shot 2021-11-16 at 5 17 10 PM](https://user-images.githubusercontent.com/13744167/141957230-d9f9ccfc-d9f6-4be3-ab60-a82bf4945089.png)


With negative margin:
![Screen Shot 2021-11-16 at 5 16 32 PM](https://user-images.githubusercontent.com/13744167/141957113-0e246d2d-5a66-4669-951f-1107b44b02a1.png)
![Screen Shot 2021-11-16 at 5 16 52 PM](https://user-images.githubusercontent.com/13744167/141957172-73c357c8-ee21-4a35-82c3-3979720857b3.png)

What do you think?

Reply:
Oke that makes sense.